test/check/check.go

```diff
@@ -113,6 +114,17 @@ func CheckConfigVersions(operatorVersion, observedVersion, targetVersion string,
 	}
 }
 
+func CheckOperatorIsReady(timeout time.Duration) {
+	By("Checking that the operator is up and running")
+	if timeout != CheckImmediately {
+		Eventually(func() error {
+			return checkForDeployment(components.Name, components.Namespace)
+		}, timeout, time.Second).ShouldNot(HaveOccurred(), fmt.Sprintf("Timed out waiting for the operator to become ready"))
+	} else {
+		Expect(checkForDeployment(components.Name, components.Namespace)).ShouldNot(HaveOccurred(), "Operator is not ready")
```

Comment:
we can do a timeout=1 and interval=1 so it tries only once and wait for 1 so the only thing we do is play with timeout and interval but reuse the assert.

Reply:
It is consistent with the other checks. Opened an issue to address this in another PR https://github.com/kubevirt/cluster-network-addons-operator/issues/167